src/DependencyInjection/ArrayConfig.php

```diff
@@ -60,9 +75,9 @@ public function getIncludePaths(): array
     /**
      * {@inheritdoc}
      */
-    public function cwd(): string
+    public function getProjectRoot(): string
     {
-        return $this->cwd;
+        return $this->project_root;
```

Comment:
Is this the root folder name / path ? How about `project_root_path` to be more specific?

Reply:
This name is consistent with the `getProjectRoot` function in the `ConfigInterface` inside the asset-lib.